extensions/nns/build.rs

```diff
@@ -1,7 +1,7 @@
 use std::env;
 use std::path::PathBuf;
 
-const REPLICA_REV: &str = "f99495f3772d5a85d25ef5008179b49a5f12c5c2";
+const REPLICA_REV: &str = "8dd5cd00acdc4d891fa0c5a1605bcf3b68a3d707";
```

Comment:
Should this commit match the one in the changelogs?

Reply:
Yes it should. Head slap